pkgs/os-specific/linux/pcm/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, callPackage, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "201710";
+  name = "pcm-${version}";
+
+  src = fetchFromGitHub {
+    owner = "opcm";
+    repo = "pcm";
+    rev = "${version}";
+    sha256 = "02rq8739zwwbfrhagvcgf6qpmnswxl9b0qsld26rg6zp91v2npbj";
+  };
+
+  nativeBuildInputs = [ ];
+  buildInputs = [ ];
+
+  buildPhase = ''
+    make
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp pcm*.x $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Processor counter monitor";
+    homepage = https://www.intel.com/software/pcm;
+    license = licenses.bsd3;
+    platforms = platforms.linux;
```

Comment:
Would you like to become the maintainer of this package? In this case, could you add yourself in the maintainers list?

Reply:
Done :)